ambry-api/src/test/java/com.github.ambry/rest/RestRequestMetricsTrackerTest.java

```diff
@@ -165,34 +163,29 @@ protected void compareMetrics(String metricPrefix, MetricRegistry metricRegistry
         histograms.get(metricPrefix + RestRequestMetrics.SC_RESPONSE_PROCESSING_WAIT_TIME_SUFFIX).getSnapshot()
             .getValues()[0]);
 
-    assertEquals("Request total CPU time unequal", totalTime,
-        histograms.get(metricPrefix + RestRequestMetrics.TOTAL_CPU_TIME_SUFFIX).getSnapshot().getValues()[0]);
     assertEquals("Rate metric has not fired", 1,
```

Comment:
Nevermind, actually.  I see that you changed the last message because failedCount can be 1 or 0.